TestProviders/ReadyProvider.psm1

```diff
@@ -54,8 +54,9 @@ Class ReadyProvider : TestProvider
 			$count = 0
 			foreach ($vmData in $AllVMData) {
 				# get the first active nic device's IPv4 address. This is the temporary approach, as it may not be right for VMs that have multi nic devices.
+				# For devices with multiple interfaces, we ignore the interface present in staticipconfig.log
 				$ipAddrInfo = Run-LinuxCmd -username $global:user -password $global:password -ip $($vmData.PublicIp) -port $($vmData.SSHPort) `
```

Comment:
Dont include the changes directly here. Add a code to differentiate the special case handling in Ready platform.